tm2/pkg/bft/privval/signer/remote/server/options.go

```diff
@@ -0,0 +1,134 @@
+package server
+
+import (
+	"fmt"
+	"log/slog"
+	"net"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/gnolang/gno/tm2/pkg/bft/types"
+	"github.com/gnolang/gno/tm2/pkg/crypto/ed25519"
+	osm "github.com/gnolang/gno/tm2/pkg/os"
+)
+
+// RemoteSignerServer provides a service that forwards requests to a types.Signer.
+type RemoteSignerServer struct {
```

Comment:
Why define this in `options.go`?

Reply:
For consistency with https://github.com/gnolang/gno/pull/3554#discussion_r2003507074